standalone/index.js

```diff
@@ -0,0 +1,43 @@
+import { Component } from '@wordpress/element';
+import '@wordpress/data';
+import '@wordpress/core-data';
+/**
+ * TODO: core-blocks in are throwing an error because this is undefined. This can happen in strict mode, since that
+ * safeguards against accidental assignment to window, when calling a class init incorrectly
+ *
+ * this.wp=this.wp||{},this.wp.coreBlocks=function(e){
+ *
+ * See where this code is generated. I can't find it in the src code, likely generated from a script?
+ *
+ */
+import { registerCoreBlocks } from '@wordpress/core-blocks';
+import { EditorProvider, WritingFlow, ObserveTyping, BlockList } from '@wordpress/editor';
+
+const editorSettings = {};
+const wpApiSettings = {
+	schema: {
+		routes: {},
+	},
+};
+const post = {
+	type: 'post',
+	content: {},
+};
+
+registerCoreBlocks();
```

Comment:
There is a number of blocks which has server-side registered definitions which might not work in this setup like Latest posts. For unit tests we generate this stub: https://github.com/WordPress/gutenberg/blob/master/core-blocks/test/server-registered.json
I'm not quite sure how it would operate in the standalone version. We probably would have to skip them at all, because they require some PHP handling anyways.